<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContextDialog</class>
 <widget class="QDialog" name="ContextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Context Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/png/org.genivi.DLTViewer.png</normaloff>:/icons/png/org.genivi.DLTViewer.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Id:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="idLineEdit">
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="descriptionLineEdit"/>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Log Level:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="loglevelComboBox">
     <item>
      <property name="text">
       <string>default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fatal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>error</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>warn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>info</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>debug</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>verbose</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Trace Status:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="tracestatusComboBox">
     <item>
      <property name="text">
       <string>default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>off</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>on</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="checkBoxUpdate">
     <property name="text">
      <string>Update data in DLT if online</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ContextDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ContextDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
